db/models/account.py

```diff
@@ -4,6 +4,16 @@
 from sqlalchemy_utils.types import UUIDType  # noqa
 
 
+import enum
+
+
+class Role(enum.Enum):
+    LEAD_ASSESSOR = "Lead Assessor"
```

Comment:
Just a curiosity, why use string value and not auto() ints as: 
![image](https://user-images.githubusercontent.com/95699325/201664294-689d3885-8b02-45eb-8faa-f273274201ae.png)

Reply:
I guess an advantage of using a defined string value is you can use the attribute value in places without any further processing, like if you wanted to display the account role to the user you could use the account.role.value to show "Lead Assessor" rather than LEAD_ASSESSOR or "1" (or "lead_assessor" which would be the value output of a StrEnum I think)?

Reply:
okay thanks, given that good reason we could probably use this pattern on other Enums 👍